src/App.js

```diff
@@ -1 +1,19 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Signup from './components/signup/signup';
```

Comment:
The names of your components should use `PascalCase`. Example: `SignUp`, and it's file name should be `SignUp.jsx`.
Do same for all your components.